uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java

```diff
@@ -818,6 +836,23 @@ public synchronized void handleLiveInstanceChange(boolean onlyCheckOffline) thro
     }
   }
 
+  private boolean isOnline(String topicName, String partition) {
```

Comment:
better method naming? Also why it's topic/partition?

Reply:
Changed to `isControllerOnline`